tests/ProjectedCameraTests.cpp

```diff
@@ -0,0 +1,108 @@
+#define _USE_MATH_DEFINES
+
+#include "Fixtures.h"
+#include "UsgsAstroProjectedSensorModel.h"
+#include "UsgsAstroPlugin.h"
+#include "UsgsAstroPluginSupport.h"
+
+#include <gtest/gtest.h>
+#include <nlohmann/json.hpp>
+
+#include <math.h>
+#include <iostream>
+
+using json = nlohmann::json;
+
+TEST_F(ConstVelocityProjectedSensorModel, ConstructFromState) {
+  std::string modelState = sensorModel->getModelState();
+  csm::RasterGM * model = getUsgsCsmModelFromState(modelState, UsgsAstroProjectedSensorModel::_SENSOR_MODEL_NAME, NULL);
+
+  // When this is different, the output is very hard to parse
+  // TODO implement JSON diff for gtest
```

Comment:
Is this something that needs to be done before merge, or just a QOL thing for debugging?

Reply:
This was a comment that was ripped from the line scan camera test that this was copied from. I can remove it just so we don't duplicate it